pyproject.toml

```
@@ -60,3 +60,9 @@ exclude = '''
   | dist
 )/
 '''
+
+[tool.pytest.ini_options]
+markers = [
+    "devs: test devnet (deselect with '-m dev')",
```

typo, and should mention selecting it, as its disabled by default

```suggestion
    "dev: test devnet (select with '-m dev')",
```

---

The pyproject.toml doesn't need an update after this?

---

nope, this is in the pyproject.toml :)
the addopts is correctly filtering out tests marked with `dev` not `devs`